code/src/cljs/sixsq/nuvla/ui/edges/views.cljs

```diff
@@ -779,13 +776,139 @@
         :i-per-page-multipliers [1 2 4]}])))
 
 
+(defn- get-name-as-keyword
+  [tr q-key]
+  (tr [(-> q-key name keyword)]))
+
+(defn- TagsEditModeRadio
+  [edit-mode opened-modal change-mode]
+  (let [tr               (subscribe [::i18n-subs/tr])
+        active?          (= opened-modal edit-mode)
+        font-weight      (if active? 700 400)
+        setting-tags?    (= spec/modal-tags-set-id edit-mode)]
+    [ui/Radio {:style     {:font-weight font-weight}
+               :label     (str (get-name-as-keyword @tr edit-mode)
+                               (when setting-tags? (str " (" (@tr [:tags-overwrite]) "!)")))
+               :checked   active?
+               :on-change #(change-mode edit-mode)}]))
+
+
+(defn- ButtonAskingForConfirmation
```

Comment:
We should extract this button to make it a reusable component. Use it in modal danger. Can be done later